components/SiteNavbar.vue

```diff
@@ -41,32 +48,38 @@
       <div
         v-show="lg || mobileNavIsActive"
```

Comment:
```suggestion
        v-if="lg || mobileNavIsActive"
```

For some reason, this is no longer working, so it always renders with the mobile nav open like this:
![image](https://user-images.githubusercontent.com/17736056/129251546-6579707c-ce6a-47ca-bc02-66025f6d2d7a.png)

Reply:
Fixed it, please avoid doing breakpoint logic in javascript when it can be done with css instead, using tailwind modifiers instead.